src/main/java/androidx/content/Context.kt

```diff
@@ -91,3 +95,25 @@ inline fun Context.withStyledAttributes(
         typedArray.recycle()
     }
 }
+
+/**
+ * Return a [Bitmap] decoded from [Resources]
+ *
+ * @param resId Resource id of a drawable from `res/drawable` folder
+ * @param options Decoding options
+ */
+inline fun Resources.getBitmap(
```

Comment:
would be cool when we get inline classes in the future: DrawableInt.getBitmap(resources) 😸